requirements.bazel.txt

```diff
@@ -6,13 +6,11 @@ protobuf>=3.5.0.post1
 six>=1.10
 wheel>=0.29
 futures>=2.2.0
-google-auth>=1.17.1
+google-auth>=1.17.2
 oauth2client==4.1.0
 requests>=2.14.2
 urllib3>=1.23
 chardet==3.0.4
 certifi==2017.4.17
 idna==2.7
 googleapis-common-protos==1.5.5
-# rsa 4.3 is the last version support Python 2
-rsa==4.3
```

Comment:
I'm pretty sure this will break. IIRC, `rules_python` requires you to identify versions for your entire dependency tree.

Reply:
I thought so, but `rsa` is a new dependency I manually added in https://github.com/grpc/grpc/pull/23197. If it wasn't breaking in the past, reverting should not break other stuff.

However, Python tests are still failing. I'm not sure where the `rsa==4.0` slipped into our CI.

Reply:
Huh. Maybe `google-auth` only has a soft dependency on `rsa`?

Reply:
Nice finding!